README.md

```diff
@@ -155,6 +175,27 @@ Try out the command below.
 $ curl -X POST localhost:8000/v2/models/vllm_model/generate -d '{"text_input": "What is Triton Inference Server?", "parameters": {"stream": false, "temperature": 0}}'
 ```
 
+Upon success, you should see a response from the server like this one:
+```
+{"model_name":"vllm_model","model_version":"1","text_output":"What is Triton Inference Server?\n\nTriton Inference Server is a server that is used by many"}
+```
+
+In the [samples](samples) folder, you can also find a sample client,
+[client.py](samples/client.py) which uses Triton's
+[asyncio gRPC client library](https://github.com/triton-inference-server/client#python-asyncio-support-beta-1)
+to run inference on Triton.
+
+### Running the Latest vLLM Version
+
+To see the version of vLLM in the container, see the
+[version_map](https://github.com/triton-inference-server/server/blob/85487a1e15438ccb9592b58e308a3f78724fa483/build.py#L83)
+in [build.py](https://github.com/triton-inference-server/server/blob/main/build.py)
+for the Triton version you are using.
+
+If you would like to use a specific vLLM commit or the latest version of vLLM, you
```

Comment:
I think this is no longer necessary.
To get the latest vLLM, you can do: `pip install git+https://github.com/vllm-project/vllm.git`
To get a specific vLLM commit, you can do something like: `pip install git+https://github.com/vllm-project/vllm.git@7d761fe3c12e87df37383467c43c97dec2bb8470`